### ISSUE TITLE
Button contrast fails in dark mode for Dialog Cancel Button

### ISSUE THREAD

**Issue by happymacarts** [PH7-Jack/livewire-ui#656]
**Describe the bug**
on dark mode the dialog button text is black over 31,41,55 and fails SiteImprove Accessibility Contrast minimum

**To Reproduce**
installed everything load the dialog component and view in dark mode

**Expected behavior**
The default button state should meet minimum Accessibility standards of 4.5

**Screenshots or Videos**
![image](https://user-images.githubusercontent.com/5170850/236314716-c1b52276-75de-47f0-8b5d-a682f09662f0.png)
![image](https://user-images.githubusercontent.com/5170850/236316413-94852954-aa53-4527-92a8-4c08d63d4e8e.png)

I ended up publishing the view and adding the class but thought this should be a default
```<div x-show="dialog && dialog.reject" x-ref="reject" class="dark:text-gray-300"></div>```

**Comment by joaopalopes24** [comment]
Thank you for using WireUi!

We released stable version 2, with new features and improvements. Check if the mentioned problem has been resolved, if not, reopen this issue and we will provide more information. Thank you in advance, we are organizing Issues for better maintenance.
